File: eTuristickaAgencija.Models/Karta.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTuristickaAgencija.Models
{
    public class Karta
    {
        public int Id { get; set; }
        public int? TerminId { get; set; }
        public int? KorisnikId { get; set; }
        public DateTime DatumKreiranja { get; set; }
        public bool Ponistena { get; set; }
    }
}

File: UI/eturistickaagencija_admin/lib/models/karta.dart
import 'package:json_annotation/json_annotation.dart';

part 'karta.g.dart';

@JsonSerializable()
class Karta{
int? id;
int? terminId;
int? korisnikId;
DateTime? datumKreiranja;
bool? ponistena;

Karta(this.id,this.terminId,this.korisnikId,this.datumKreiranja,this.ponistena);

factory Karta.fromJson(Map<String,dynamic>json)=>_$KartaFromJson(json);
Map<String,dynamic>toJson()=>_$KartaToJson(this);
}
